Makefile.config

```diff
@@ -31,7 +31,7 @@
 
 # 1. Use ROOT or not (never used on MIC)
 # Comment out to disable root ("1" is not relevant)
-#WITH_ROOT := 1
+WITH_ROOT := 1
```

Comment:
@davidlange6 please revert this, ROOT defaults shouldn't be changed

Reply:
whoops - my mistake